### PR TITLE
Deploying High Level Pytest Infra cloud api tests

### PR DESCRIPTION
**Pull request by abukhoy**
This the follow up to #25 (there were some issues with the PR)
In this PR, we have added support of HL API testing script. 

**Note**: For more details, see tests/README.md

### CODE REVIEW

**Comment by vbaddi** [comment]
cc @ochougul

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Required Plugins
+- `pytest`
+- `pytest-mock`
+
+You can install them using pip:
+```sh
+pip install pytest pytest-mock
+```
+Alternatively, if you have specefied these dependencies in your `setup.py` or `setup.cfg` , you can install them using the extras_require feature:
```
Comment (ochougul) [inline tests/README.md:25]: this has been moved to `pyproject.toml` now

Comment (abukhoy) [reply tests/README.md:25]: Okay, replacing it now.

**Comment by abukhoy** [comment]
`tests/README.md` has been changed now.

**Review by ochougul — APPROVED** [review]

**Review by vbaddi — APPROVED** [review]

**Pull request merged**